src/test/MAS_tests.cc

```diff
@@ -1,9 +1,78 @@
 #include <gtest/gtest.h>
+#include "../alignment/State.h"
+#include "../alignment/Edge.h"
+#include "../alignment/Node.h"
+#include "../alignment/Graph.h"
 
 namespace MAS_test {
     TEST(MAS_tests, test_test) {
         EXPECT_EQ(0,0);
     }
+// Timon: score_test
+    TEST(MAS_tests, test_test) {
```

Comment:
Please change the arguments of the TEST marco.
* first argument is the suite name, which could be "alignment_tests" in this case
* second argument is the case name and can be replaced with "score_test"